src/main/java/core/basesyntax/data/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.data.dao;
+
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String INPUT_DATA = "fruit_shop_transactions.csv";
```

Comment:
The constant `INPUT_DATA` is not used anywhere in the code. Consider removing it to avoid confusion and adhere to the checklist's advice to avoid hardcoding values outside of the Main class.